src/components/atoms/BadgePreview/BadgePreviewProps.ts

```diff
@@ -22,4 +22,6 @@ export interface BadgePreviewProps {
   miniIcon?: React.ReactNode // Icon above the title
   selected?: boolean // Marks the badge as selected
 }
-export type MiniBadgePreviewProps = Omit<BadgePreviewProps, 'size' | 'badgeUrl' | 'iconUrl'>
+export type MiniBadgePreviewProps = Omit<BadgePreviewProps, 'size' | 'badgeUrl' | 'iconUrl' | 'description'> & {
+  description: string
```

Comment:
why omiting the description and then adding the description again but restricting only as string?
We could have also a component description for the mini badge, perhaps we don't need it yet but it does not bother if it already supports it